src/test/java/edu/hawaii/its/groupings/access/AuthorizationServiceTest.java

```diff
@@ -99,6 +99,7 @@ public void fetchPrivilegesTest() {
         assertThat(roleHolder.size(), is(4));
         assertTrue(roleHolder.contains(Role.ANONYMOUS));
         assertTrue(roleHolder.contains(Role.UH));
+        assertFalse(roleHolder.contains(Role.DEPARTMENT));
```

Comment:
In the lines above, there are mocks that fake a function to return true. Could you mock the isDepartmentAccount() function to return true then assertTrue that roleHolder.contains(Role.DEPARTMENT). This is just to match the rest of the tests to assertTrue. Thanks.

Reply:
I reassigned the value for uhUuid in order to use assertTrue in this part of the test.
```
uhUuid = "test";
roleHolder = authorizationService.fetchRoles(uhUuid, "test");
assertTrue(roleHolder.contains(Role.DEPARTMENT));